iot/index.html.slim

```diff
@@ -13,8 +13,8 @@ p
   a[href="https://community.jboss.org/en/iot/dev"]
     |  this forum for discussing developing/modifying projects within IoT environments
   | ; we'll work to answer questions or re-direct where appropriate.
-p  
-  | At some point we will add some text that describes the IoT. For now we will assume that the reader understands basic concepts of IoT and further details can be obtained from the 
+p
+  | At some point we will add some text that describes the IoT. For now we will assume that the reader understands basic concepts of IoT and further details can be obtained from the
```

Comment:
I originally pulled this from Gregoire's page.  I'll fix the grammar and keep it short.